File: web3rpc/rpc-specs/code-samples/python/klay/filter/uninstallFilter.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

quantity = "0xd32fd16b6906e67f6e2b65dcf48fc272"

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.uninstall_filter(quantity)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/filter/uninstallFilter.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const quantity = '0xd32fd16b6906e67f6e2b65dcf48fc272'

    sdk.klay.uninstallFilter(quantity, {}, (err, data, response) => {
        console.log(data);
    });
}
)()